src/pages/index.js

```diff
@@ -46,9 +91,50 @@ const imagePopup = new PopupWithImage(".popup-photo");
 imagePopup.setEventListeners();
 
 /* ------------------------------- Card Popup ------------------------------- */
+
+function handleDeleteCard(card) {
+  confirmationPopup.open(card);
+  confirmationPopup.setSubmitAction(() => {
+    api
+      .deleteCard(card._id)
+      .then(() => {
+        card._handleRemoveCard(this);
+        confirmationPopup.close();
+      })
+      .catch((err) => {
+        console.error(err);
+      });
+  });
+}
+
+function handleLikeButton(card) {
+  if(!card.isLiked) {
+    api.likeCard(card._id)
+     .then(() => {
+        card.isLiked = true;
+        card._likeButton.classList.add("elements__button_active");
```

Comment:
`        card._likeButton.classList.add("elements__button_active");
`

Now only the `Card` class knows every **card selector**, so you need to encapsulate the likes logics inside its methods and call them in `index.js`. It’s a violation of **Single responsibility** principle


```js

        card.handleLikeIcon()

```